packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -370,3 +388,7 @@ export interface AccountHistoryCountOptions {
   txtype?: TxType | string
   no_rewards?: boolean
 }
+
+export interface Address {
+  [key: string]: AccountRegexType[]
+}
```

Comment:
```suggestion
export interface AddressBalances {
  [key: string]: AccountRegexType[]
}
```

`Address` is quite misleading, how about `AddressBalances`?

Reply:
Okay.